### PR TITLE
chore: artist ids filtering to collectedArtistsConnection [WIP]

### PR DESCRIPTION
**Pull request by MounirDhahri**
add artist ids filtering to `collectedArtistsConnection` 

Blocked by https://github.com/artsy/gravity/pull/16525

### CODE REVIEW

**Review by olerichter00 — APPROVED** [review]
Looks good! Nice tests ✨

**Comment by olerichter00** [comment]
I've merged https://github.com/artsy/gravity/pull/16525 🚂

File: src/schema/v2/me/myCollectionInfo.ts [hunk 1]
```diff
@@ -163,14 +163,17 @@ export const myCollectionInfoFields = {
     description: "A connection of artists in the users' collection",
     type: artistConnection.connectionType,
     args: pageable({
-      sort: ArtistSorts,
-      page: { type: GraphQLInt },
-      size: { type: GraphQLInt },
+      artistIDs: {
+        type: new GraphQLList(GraphQLString),
```
[the remark this answers is not in the document]
Comment (mzikherman) [reply src/schema/v2/me/myCollectionInfo.ts:167]: Can also add a non-null constraint to the string elements, so:`type: new GraphQLList(new GraphQLNonNull(GraphQLString))`

**Review by mzikherman — APPROVED** [review]

File: src/schema/v2/me/myCollectionInfo.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (MounirDhahri) [reply src/schema/v2/me/myCollectionInfo.ts:167]: done ✅ 
done ✅

**Comment by olerichter00** [comment]
Please do not merge this PR now. I've made some changes to the resolver in https://github.com/artsy/metaphysics/pull/5133 and would like to discuss them first 🙏

**Comment by olerichter00** [comment]
I suppose, we can close this PR, since collectedArtistsConnection is deprecated in favor of userInterestsConnection.

**Comment by MounirDhahri** [comment]
Yess

**Pull request closed without merging**